dev/gen_protos.sh

```diff
@@ -5,7 +5,7 @@ if ! cargo install --list | grep "protoc-gen-prost-crate" > /dev/null; then
         exit 1
     fi
 fi
-if ! buf generate https://github.com/xmtp/proto.git#branch=xmtpv3,subdir=proto; then
+if ! buf generate https://github.com/xmtp/proto.git#branch=nmolnar/mls-verification-service,subdir=proto; then
```

Comment:
Will switch this back to `main` once we merge https://github.com/xmtp/proto/pull/107